scripts/globals/effects/soul_enslavement.lua

```diff
@@ -4,12 +4,16 @@
 local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
+    target:addMod(xi.mod.ENSPELL, 22)
```

Comment:
How does this interact with other enspells like endark?

Reply:
Good question, actually. I don't know.

Time to hit retail and find out. I can leave a TODO in the meantime

Reply:
Not sure if you ever revisited this but I'm leveling DRK tonight so I'll test this out.